atom/browser/lib/power_observer_linux.cc

```diff
@@ -100,6 +136,22 @@ void PowerObserverLinux::OnPrepareForSleep(dbus::Signal* signal) {
   }
 }
 
+void PowerObserverLinux::OnPrepareForShutdown(dbus::Signal* signal) {
+  dbus::MessageReader reader(signal);
+  bool status;
+  if (!reader.PopBool(&status)) {
+    LOG(ERROR) << "Invalid signal: " << signal->ToString();
+    return;
+  }
+  if (status) {
```

Comment:
Sorry for nitpicking, but `status` is a really bad name for a boolean variable.
It's hard to guess that `true` or `false` values might mean.

Reply:
How about using `active` as shown in the [docs](https://www.freedesktop.org/wiki/Software/systemd/inhibit/)? I'm not sure if that name is better though.

Reply:
`active` is better since it implies a binary state of active/inactive, while `status` can have numerous values, e.g. http status codes.

Reply:
:+1: